config/overlays/featuregate/webhook-provider-certmanager/kustomization.yaml

```diff
@@ -0,0 +1,15 @@
+# kustomization file for secure OLMv1
```

Comment:
That's great, but could we change the title to clarify that we're adding more than just the documents?
Something like adding kustomize files to enable feature gates for the webhook feature and documentation.
WDYT?

Reply:
fixed